public/index.html

```diff
@@ -29,17 +29,12 @@
       }
       </script>
 
-    <title>API Docs</title>
+    <title>API Catalog</title>
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div class=“rhd-m-max-width-xl”>
-      <esi:include src=“https://developers.redhat.com/api/chrome/rh-universal-nav-header” />
-      <!--#include virtual="/.include/chrome/rh-universal-nav-header/rh-universal-nav-header.html" -->
```

Comment:
These "virtual includes" are actually good, only the `esi:include`'s 
Virtual includes are the `<!--#include`'s . could you put them back? Thanks!